jberet-feature-pack/common/src/main/resources/packages/org.jboss.jberet/pm/wildfly/tasks.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" ?>
+
+<tasks xmlns="urn:wildfly:wildfly-feature-pack-tasks:2.0">
+    <copy-artifact artifact="org.jberet:jberet-core" to-location="modules/system/layers/base/org/jberet/jberet-core/main/overridden/jberet-core-jar/"/>
+</tasks>
```

Comment:
I'm not sure why this would be needed.

Reply:
If I remove this, when I'm running `mvn wildfly:package`  it will throw exception like this:

<img width="3008" alt="image" src="https://github.com/jberet/jsr352/assets/201907/face48fe-9701-481d-b2e2-d71b3f8eea2a">